doc/mgr/dashboard.rst

```diff
@@ -937,30 +938,30 @@ The above role dictates that a user has *read* and *create* permissions for
 features related to pool management, and has full permissions for
 features related to RBD image management.
 
-The Dashboard already provides a set of predefined roles that we call
-*system roles*, and can be used right away in a fresh Ceph Dashboard
+The Dashboard provides a set of predefined roles that we call
+*system roles*, which can be used right away by a fresh Ceph Dashboard
 installation.
 
 The list of system roles are:
 
-- **administrator**: provides full permissions for all security scopes.
-- **read-only**: provides *read* permission for all security scopes except
-  the dashboard settings.
-- **block-manager**: provides full permissions for *rbd-image*,
+- **administrator**: allows full permissions for all security scopes.
+- **read-only**: allows *read* permission for all security scopes except
+  dashboard settings.
+- **block-manager**: allows full permissions for *rbd-image*,
   *rbd-mirroring*, and *iscsi* scopes.
-- **rgw-manager**: provides full permissions for the *rgw* scope
-- **cluster-manager**: provides full permissions for the *hosts*, *osd*,
+- **rgw-manager**: allows full permissions for the *rgw* scope
+- **cluster-manager**: allows full permissions for the *hosts*, *osd*,
   *monitor*, *manager*, and *config-opt* scopes.
-- **pool-manager**: provides full permissions for the *pool* scope.
-- **cephfs-manager**: provides full permissions for the *cephfs* scope.
+- **pool-manager**: allows full permissions for the *pool* scope.
+- **cephfs-manager**: allows full permissions for the *cephfs* scope.
 
-The list of currently available roles can be retrieved by the following
+The list of vailable roles can be retrieved by the following
 command::
 
   $ ceph dashboard ac-role-show [<rolename>]
 
-It is also possible to create new roles using CLI commands. The available
-commands to manage roles are the following:
+It is also possible to create new roles using the CLI. The available
```

Comment:
You can also use the CLI to create new roles.

Reply:
ack